conftest.py

```diff
@@ -0,0 +1,73 @@
+import logging
+import os
+from typing import Any
+
+import pytest
+import requests
+
+import openeo
+from openeo.capabilities import ComparableVersion
+
+_log = logging.getLogger(__name__)
+
+os.environ["ENDPOINT"] = "https://openeo.dataspace.copernicus.eu/"
```

Comment:
This still has to be removed.

and, e.g. the alternative should be documented in a readme